apps/api/src/app/shared/commands/base.command.ts

```diff
@@ -1,13 +1,12 @@
-import { ClassConstructor, plainToClass } from 'class-transformer';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { plainToClass } from 'class-transformer';
 import { validateSync } from 'class-validator';
 import * as Sentry from '@sentry/node';
 import { BadRequestException, flatten } from '@nestjs/common';
 
-export class CommandHelper {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
-  static create<T>(command: ClassConstructor<T>, data: any): T {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const convertedObject = plainToClass<T, any>(command, {
+export abstract class BaseCommand {
```

Comment:
- renamed `CommandHelper` to `BaseCommand`.
- added the `abstract` keyword so it won't be possible to create the instance from just this class.
Improved the `create` function:
- `this` keyword in the static function is pointing to the class itself (constructor), for ex. if the `LoginCommand` is extending the `BaseCommand` then `this` will point to the `LoginCommand` class
- improved the type safety: TypeScript will check if the type of passed `data` object is matching the class